src/Components/RelatedCollectionsRail/index.tsx

```diff
@@ -0,0 +1,53 @@
+// import { ArtistCollectionsRailQuery } from "__generated__/ArtistCollectionsRailQuery.graphql"
+import { RelatedCollectionsRailQuery } from "__generated__/RelatedCollectionsRailQuery.graphql"
+import { SystemContextConsumer } from "Artsy"
+import React from "react"
+import { graphql, QueryRenderer } from "react-relay"
+import { RelatedCollectionsRailFragmentContainer as RelatedCollectionsRail } from "./RelatedCollectionsRail"
+
+interface Props {
+  category: string
+  showOnEditorial?: boolean
+}
+
+export const RelatedCollectionsRailContent: React.SFC<Props> = passedProps => {
+  return (
+    <SystemContextConsumer>
```

Comment:
We should be using hooks where we can now, which flattens things out quite nicely:

```tsx
import React, { useContext } from 'react'
import { SystemContext } from 'Artsy'

...

export const RelatedCollectionsRailContent: React.SFC<Props> = props => {
  const { relayEnvironment } = useContext(SystemContext)
  
  return ( ... ) 
}
```

Reply:
Or even better, rather than pulling from our context manually, we can create a new hook inside of `src/Artsy` called `useSystemContext` that we reference instead:

```tsx
import { useContext } from 'react'
import { SystemContext } from 'Artsy'

export const useSystemContext() {
  const systemContext = useContext(SystemContext) 
  return  systemContext
}
```
Then later
```tsx
import { useSystemContext } from 'Artsy'

...

const { relayEnvironment } = useSystemContext()
```

Reply:
I opted to grab the `relayEnvironment` from the `useContext` api. I was thinking the later suggestion could be addresses in a separate PR.

Reply:
@xtina-starr - Would you mind taking care of this in another PR and letting #front-end know about the new hook once it's merged?

Reply:
Will do!